content/docs/0.8.x/monitoring/prometheus/install/index.md

```diff
@@ -13,33 +13,72 @@ In order to evaluate the quality gates and allow self-healing in production, we
 
 ## Setup Prometheus
```

Comment:
Can you please change it to: `## Setup Prometheus Keptn integration`

Reply:
Done